frontend/public/locales/en/translation.json

```
@@ -162,12 +162,15 @@
     "bareMetalAsset.statusMessage.UnexpectedResourceCount": "Unexpected number of resources found on SyncSet",
     "bareMetalAsset.tableHeader.address": "Controller address",
     "bareMetalAsset.tableHeader.cluster": "Cluster",
+    "bareMetalAsset.tableHeader.hostname": "Hostname",
```

instead of keys, lets use the value as key too..
```suggestion
    "Hostname": "Hostname",
```

applies for all others